src/components/onboarding/need-wallet/index.js

```diff
@@ -0,0 +1,87 @@
+/** @jsx jsx */
```

Comment:
I'd rename this scene to `get-wallet`. But not critical, up to you.

Reply:
changed and all routes updated.